src/test/java/uk/gov/pay/api/it/CreatePaymentITest.java

```diff
@@ -51,6 +52,31 @@
     private static final String SUCCESS_PAYLOAD = paymentPayload(AMOUNT, RETURN_URL, DESCRIPTION, REFERENCE);
     private static final String GATEWAY_TRANSACTION_ID = "gateway-tx-123456";
 
+    @Test
+    public void createCardPaymentWithEmptyMetadata() {
```

Comment:
Could this maybe be called something like `createCardPaymentWithEmptyMetadataDoesNotStoreMetadata` so the name summarises the both what happens and the expected result?